includes/templates/template_default/templates/tpl_account_history_info_default.php

```diff
@@ -112,6 +112,17 @@
   if (DOWNLOAD_ENABLED == 'true') require($template->get_template_dir('tpl_modules_downloads.php',DIR_WS_TEMPLATE, $current_page_base,'templates'). '/tpl_modules_downloads.php');
 ?>
 
+<?php
+  $additional_content = false;
+  $zco_notifier->notify('NOTIFY__INVOICE_ADDITIONAL_DATA_MIDDLE', $order, $additional_content);
```

Comment:
Was this intended to have those double-underscores?  Shouldn't it be `NOTIFY_INVOICE_ADDITIONAL_DATA_MIDDLE`?